source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -138,56 +176,87 @@ InjaTransformer::InjaTransformer(
     }
     break;
   }
-  case envoy::api::v2::filter::http::TransformationTemplate::
-      kMergeExtractorsToBody: {
+  case TransformationTemplate::kMergeExtractorsToBody: {
     merged_extractors_to_body_ = true;
     break;
   }
-  case envoy::api::v2::filter::http::TransformationTemplate::kPassthrough:
+  case TransformationTemplate::kPassthrough:
     break;
-  case envoy::api::v2::filter::http::TransformationTemplate::
-      BODY_TRANSFORMATION_NOT_SET: {
+  case TransformationTemplate::BODY_TRANSFORMATION_NOT_SET: {
     break;
   }
   }
+
+  // parse environment
+    for (char **env = environ; *env != 0; env++) {
```

Comment:
Should this maybe be optional? Not sure how much that would save, but this code is totally unecessary 99% of the time

Reply:
this only happens one time, i.e. it's not in the request path

Reply:
oh duh